withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -126,11 +145,14 @@ Unless otherwise noted, all values will be passed as double-quote encapsulated c
   }
 }
 ```
+
 **Results detail:**
- - success: Was the attempt successful? (boolean)
- - memoMode: Is the system in MemoMode? (boolean)
+
+- success: Was the attempt successful? (boolean)
+- memoMode: Is the system in MemoMode? (boolean)
 
 **When error condition is encountered:**
+
 ```json
 {
   "errorCode": 999,
```

Comment:
this new line is needed "displayErrorMessage": ["display error message detail"] , right after Line "loggingErrorMessage": "error message detail",

Reply:
Fixed.